JavaScript/chapter01/1.5 - OneAway/rroque98_sol.js

```diff
@@ -18,19 +19,32 @@ const isOneAway = (str1, str2) => {
     let errorCount = 0;
     const longestStr = findLongestStr(str1, str2);
     let x = 0;
-    for(let i = 0; i < longestStr; i++) {
-      if (str1[i] !== str2[x]) {
-        errorCount++;
+    for(let i = 0; i < longestStr.length; i++) {
+      let currentChar1 = str1[i];
+      let currentChar2 = str2[x]
+      let nextChar1 = str1[i + 1];
```

Comment:
it's not completely obvious to a random person reading your code why `i+1` or `x+1` are safe (actually i'm not even sure myself).

A few alternatives:

a) modify the loop condition so that it takes this into account:
```javascript
for (let i = 0, x = 0; i + 1 < str1.length && x + 1 < str2.length; i++) {
```
b) Add a comment with a rough "proof" of why this will never ever crash.